openapi/dali-upload.yaml

```diff
@@ -0,0 +1,7 @@
+name: UPLOAD
+in: query
+description: |
+  specify comma-separated name,location pair for a temporary table to be 
+  uploaded and used in the request
+schema:
+    type: string
```

Comment:
`pattern:` might also be useful here to enforce some simple regex for the comma separation.

Reply:
I think `\w` is roughly correct for valid table names... although it does not capture the fact that the name must start with an letter. I should fix that.

That regex101.com site is great - thnx for that pointer!